src/integration/integration_suite_test.go

```diff
@@ -93,9 +92,7 @@ var _ = SynchronizedAfterSuite(func() {
 })
 
 var _ = BeforeEach(func() {
-	httpClient = cfhttp.NewCustomTimeoutClient(httpRequestTimeout)
-	startApiServer()
-	startServiceBroker()
```

Comment:
Since you moved the starts into the respective tests, I would move the stopAll into each one as well. It is very confusing to have it stop out here but start inside.